### ISSUE TITLE
Issue with SVG and PNG Icons Not Loading on iPhone in .NET8/Uno 4.10.34

### ISSUE THREAD

**Issue by NerijusN** [unoplatform/uno#17070]
### Current behavior

In .NET7, our application did not encounter any issues loading PNG icons. However, after updating to .NET8 and Uno 4.10.34, PNG icons stopped loading. We attempted to convert our icons to SVG paths, but the problem persists, specifically on iPhone. The same code works perfectly on Android and Windows.

If we have definitions (we tried both ways with no result on iPhone):
```
<x:String x:Key="IconPath.List" >M 0 0 h 2 v 2 h -2 v -2 M 0 4 h 2 v 2 h -2 v -2 M 0 8 h 2 v 2 h -2 v -2 M 4 0 h 14 v 2 h -14 v -2 M 4 4 h 14 v 2 h -14 v -2 M 4 8 h 14 v 2 h -14 v -2 Z</x:String>

<DataTemplate x:Key="IconTemplate.List">
	<Canvas Width="18" Height="10">
		<Path Data="M 0 0 h 8 v 8 h -8 v -8 M 0 16 h 8 v 8 h -8 v -8 M 0 32 h 8 v 8 h -8 v -8 M 16 0 h 56 v 8 h -56 v -8 M 16 16 h 56 v 8 h -56 v -8 M 16 32 h 56 v 8 h -56 v -8 Z"
		      Fill="#DEE7EAEF" />
	</Canvas>
</DataTemplate>
```
Code that works on Android/Windows but not iPhone: 
```
<toolkit:CommandBarExtensions.NavigationCommand>
	<AppBarButton Command="{Binding OpenPane}">
		<AppBarButton.Icon>
			<PathIcon Data="{StaticResource IconPath.List}" Width="18" Height="10" Foreground="Red"/>
		</AppBarButton.Icon>
	</AppBarButton>
</toolkit:CommandBarExtensions.NavigationCommand>
```
Also code below results in same problems:
```
<toolkit:CommandBarExtensions.NavigationCommand>
	<AppBarButton Command="{Binding OpenPane}">
		<Viewbox Height="10"
		         Width="18"
		         VerticalAlignment="Center">
			<ContentControl ContentTemplate="{StaticResource IconTemplate.List}" />
		</Viewbox>
	</AppBarButton>
</toolkit:CommandBarExtensions.NavigationCommand>
```						
### Observations
* On iPhone, there is no icon and no red background.
* Some SVG and PNG icons are missing on iPhone.
* Icons work as expected when used in other contexts, for example:
```
<Viewbox Grid.Column="1"
         Height="140"
         Width="80"
         Margin="0 16 0 4">
	<ContentControl ContentTemplate="{StaticResource IconTemplate.Appearance.Light}" />
</Viewbox>
```
### Request for Assistance
Please provide guidance on how to make SVG paths work correctly on iPhone for .NET8/Uno 4.10.34. Specifically, we need to understand:

* If there are any known issues with PathIcon or ContentControl on iOS in this version of Uno.
* Any recommended workarounds or fixes to ensure icons render correctly across all platforms.  

Thank you for your assistance in resolving this critical issue.

### Expected behavior

SVG path works on all devices including iPhone

### Workaround

None found.

### Works on UWP/WinUI

Yes

### NuGet package version(s)

| Package                              | Version               | Condition              |
|--------------------------------------|-----------------------|------------------------|
| BiometryService                      | 1.1.0                 |                        |
| Uno.Fonts.Fluent                     | 2.3.0                 |                        |
| Uno.WinUI                            | 4.10.34               |                        |
| Uno.WinUI.RemoteControl              | 4.10.34               | '$(Configuration)'=='Debug' |
| Uno.UI.Adapter.Microsoft.Extensions.Logging | 4.10.34       |                        |
| Uno.Microsoft.Xaml.Behaviors.Interactivity.WinUI | 2.3.1-uno.2 |                        |
| Uno.Microsoft.Xaml.Behaviors.WinUI.Managed | 2.3.1-uno.2     |                        |
| Reactive.Annex.Uno.WinUI             | 1.0.2                 |                        |
| Microsoft.Extensions.Logging.Console | 8.0.0                 |                        |
| Chinook.BackButtonManager            | 1.1.4                 |                        |
| Chinook.BackButtonManager.Uno.WinUI  | 1.1.4                 |                        |
| Chinook.DataLoader.Uno.WinUI         | 1.2.0                 |                        |
| Chinook.DataLoader.DynamicMvvm       | 1.2.0                 |                        |
| Chinook.DynamicMvvm.Uno.WinUI        | 1.4.2                 |                        |
| Chinook.DynamicMvvm                  | 1.4.2                 |                        |
| Chinook.DynamicMvvm.CollectionTracking | 1.4.2               |                        |
| Chinook.DynamicMvvm.Reactive         | 1.4.2                 |                        |
| Chinook.DynamicMvvm.FluentValidation | 1.4.2                 |                        |
| Serilog.Sinks.Xamarin                | 1.0.0                 |                        |
| Uno.CodeGen                          | 2.0.0-dev.10          |                        |
| Uno.Injectable                       | 2.0.0-dev.10          |                        |
| Uno.SourceGenerationTasks            | 4.2.0                 |                        |
| MallardMessageHandlers               | 1.2.0                 |                        |
| Chinook.StackNavigation.Abstractions | 2.1.1                 |                        |
| Chinook.SectionsNavigation.Uno.WinUI | 2.1.1                 |                        |
| Chinook.SectionsNavigation.Reactive  | 2.1.1                 |                        |
| MessageDialogService.Uno.WinUI       | 1.0.2                 |                        |
| Uno.Material.WinUI                   | 2.6.1                 |                        |
| Uno.Toolkit.WinUI                    | 3.0.4                 |                        |
| Uno.Toolkit.WinUI.Material           | 3.0.4                 |                        |
| Nventive.Persistence.Reactive        | 0.4.0-dev.43          |                        |
| Nventive.Persistence.Uno.WinUI       | 0.4.0-dev.43          |                        |
| Nventive.View.Uno.WinUI              | 0.5.0-dev.74          |                        |
| ExtendedSplashScreen.Uno.WinUI       | 1.0.2                 |                        |
| Uno.WinUI.Lottie                     | 4.10.34               |                        |
| Uno.CommunityToolkit.WinUI.DeveloperTools | 7.1.100           |                        |
| Uno.CommunityToolkit.WinUI.UI.Controls | 7.1.100             |                        |
| Uno.WinUI.Svg                        | 4.10.34               |                        |
| Svg.Skia                             | 0.5.18                |                        |

### IDE

Visual Studio 2022

### IDE version

17.10.1

**Comment by jeromelaban** [comment]
Thanks for the report. 

There should not be any specific change between net7 and net8 at this time that can cause this type of behavior, even though Uno 4.10.x does not explicitly support net8. Note that we may not be able to fix this particular behavior if it requires building Uno for net8 for 4.10.x, and will require moving to Uno 5.0 or later.

If you're able to make an individual repro that demonstrates the issue, it will make a possible resolution faster.

**Comment by jeromelaban** [comment]
@NerijusN would you have update on the question above? We're otherwise still trying to backport multiple fixes from more recent releases to tentatively fix this issue on 4.10, but we're not there yet.

**Comment by jeromelaban** [comment]
@NerijusN The 4.10.39 build is now available on nuget.org. We do not know if this will fix this particular issue, as we're not sure if it reproduces on your side. Let us know if this fixes your issue. Thanks!
